doc/CMakeLists.txt

```diff
@@ -0,0 +1,131 @@
+cmake_minimum_required(VERSION 3.4)
+
+project(sml LANGUAGES CXX)
+
+find_package(Boost          REQUIRED)
+
+if(NOT MSVC)
+  # c++11 options
+  CHECK_CXX_COMPILER_FLAG(-std=c++11 HAS_CXX11_FLAG)
+  CHECK_CXX_COMPILER_FLAG(-std=c++0x HAS_CXX0X_FLAG)
+  if(HAS_CXX11_FLAG)
+    message(STATUS "Compiling with C++11 support")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+  elseif(HAS_CXX0X_FLAG)
+    #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+  endif()
+endif()
+
+add_subdirectory(benchmark)
+add_executable(euml
+  benchmark/complex/euml.cpp)
+add_executable(euml2
+  benchmark/complex/euml2.cpp)
+add_executable(sc
+  benchmark/complex/sc.cpp)
+add_executable(sml
+  benchmark/complex/sml.cpp)
+add_executable(switch
+  benchmark/complex/switch.cpp)
+add_executable(variant
+  benchmark/complex/variant.cpp)
+add_executable(euml
+  benchmark/composite/euml.cpp)
+add_executable(euml2
+  benchmark/composite/euml2.cpp)
+add_executable(sc
+  benchmark/composite/sc.cpp)
+add_executable(sml
+  benchmark/composite/sml.cpp)
+add_executable(euml
+  benchmark/header/euml.cpp)
+add_executable(euml2
+  benchmark/header/euml2.cpp)
+add_executable(sc
+  benchmark/header/sc.cpp)
+add_executable(sml
+  benchmark/header/sml.cpp)
+add_executable(euml
+  benchmark/simple/euml.cpp)
+add_executable(euml2
+  benchmark/simple/euml2.cpp)
+add_executable(sc
+  benchmark/simple/sc.cpp)
+add_executable(sml
+  benchmark/simple/sml.cpp)
+
+add_subdirectory(doc)
+add_executable(god_object
+    doc/cppnow-2017/example/god_object.cpp)
+
+add_executable(at
+  doc/cppnow-2016/example/at.cpp)
+add_executable(hana_at
+  doc/cppnow-2016/example/hana_at.cpp)
+add_executable(hana_unique
+  doc/cppnow-2016/example/hana_unique.cpp)
+add_executable(mpl_at
+  doc/cppnow-2016/example/mpl_at.cpp)
+add_executable(mpl_unique
+  doc/cppnow-2016/example/mpl_unique.cpp)
+add_executable(unique
+  doc/cppnow-2016/example/unique.cpp)
+add_executable(god_object
+  doc/cppnow-2017/example/god_object.cpp)
+add_executable(static_polymorphism
+  doc/cppnow-2017/example/static_polymorphism.cpp)
+add_executable(type_erasure
+  doc/cppnow-2017/example/type_erasure.cpp)
+add_executable(connection_v1
+  doc/embo-2017/example/connection_v1.cpp)
+# doc/embo-2017/example/connection_v2.cpp
```

Comment:
Any specific reasons for commenting those examples out?

Reply:
Quick and dirty, then forget..
It's fixed in the upcoming version.